sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -641,7 +641,8 @@ SIREPO.app.directive('logoutMenu', function(authState, authService, requestSende
               '</a>',
               '<ul class="dropdown-menu">',
                 '<li class="dropdown-header"><strong>{{ ::authState.displayName }}</strong></li>',
-                '<li class="dropdown-header" data-ng-if="::authState.userName">{{ ::authState.userName }} via {{ ::authState.method }}</li>',
+                '<li class="dropdown-header">{{ authState.planName() }}</li>',
```

Comment:
paymentPlan()

Reply:
https://github.com/radiasoft/sirepo/pull/2711/commits/110ea9c3b63ca85561bac84b1dd582a383edab18

It can't be paymentPlan because that conflicts with paymentPlan sent back from the server. I maded it paymentPlanName()